packages/x-data-grid/src/utils/createSelector.ts

```diff
@@ -91,6 +91,7 @@ export const createSelector = ((
   b: Function,
   c?: Function,
   d?: Function,
+  // eslint-disable-next-line id-denylist
```

Comment:
😄 
Ok, since this will be removed in v8, let's not waste time on this. Otherwise we could've though about naming these f1, f2, f3,...

Reply:
We have the v8 equivalent below with the exact same naming "issue" though :p